.github/workflows/deploy.yaml

```diff
@@ -4,15 +4,33 @@ on:
   pull_request:
     branches:
       - main
+    
   push:
     branches:
       - main
     # Review gh actions docs if you want to further define triggers, paths, etc
     # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#on
 
 jobs:
+  build:
```

Comment:
Let's separate the PR pipeline in another file, like the documentation examples.

Reply:
done